exercises/exercise3.js

```diff
@@ -1,12 +1,56 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Would you rather!',
+    prompt: 'Would you rather go shopping or out to eat?',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'its 2020, solid food is for the weak! lets go shopping';
     },
+    isCaseSensitive: false,
+    isReply: true,
+    isLitening: true
   },
+
+  {
+    label: 'Would you rather the aliens that make first contact be organics or robots',
+     prompt: 'I would rather #{name} make first contact. ',
+      handler: function (params) {
+        
+        if ( params.name === "robots"  ){
+          return " really?" + params.name1 + "would be my last choice"
+        }
+        else  {
+          return "See your smart!";
+        }
+      },
+      isCaseSensitive: false,
+      isReply: true,
+      isLitening: true
+    },
+      {
+        label: "Dance break",
+        prompt:"#{playerTwo} challanges #{playerOne} to a dance off, who will win?",
```

Comment:
The way that the hubot code was written on the back end it does not like or does not know how to compute a prompt that starts with or only has a #{param} without anything in front of it. So, starting with "#{playerTwo}" was causing an error in the bot load. This is easily fixed by adding in a word like "If #{playerTwo}...". Nothing to do with you and everything to do with how this bot was built on the back end.